File: FinalAssignment/app/src/main/java/com/example/finalassignment/MainActivity.java
package com.example.finalassignment;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public boolean onCreateOptionsMenu(Menu menu){
        getMenuInflater().inflate(R.menu.options,menu);
        return true;
    }

    public boolean onOptionsItemSelected(MenuItem item){
        switch (item.getItemId()){
            case R.id.insert:
                Intent intent = new Intent(MainActivity.this,registration.class);
                startActivity(intent);
                return true;
            case R.id.view:
                Toast.makeText(getApplicationContext(),"Selection is VIEW",Toast.LENGTH_LONG).show();
                return true;
            case R.id.delete:
                Toast.makeText(getApplicationContext(),"Selection is DELETE",Toast.LENGTH_LONG).show();
                return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: KotlinSecond/app/src/main/java/com/example/kotlinsecond/MainActivity.kt
package com.example.kotlinsecond

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Gravity
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textView = findViewById<TextView>(R.id.textView2)
        val editText = findViewById<EditText>(R.id.input_text)
        val button = findViewById<Button>(R.id.button)

        button.setOnClickListener(){
           /* val linearLayout = null
            val layout = layoutInflater.inflate(R.layout.custom_tag, linearLayout)
            val myToast = Toast(applicationContext)
            myToast.duration = Toast.LENGTH_LONG
            myToast.setGravity(Gravity.CENTER_VERTICAL,0,0)
            myToast.view = layout
            myToast.show()*/

            intent = Intent(this,MainActivity2::class.java)
            startActivity(intent)

        }
    }
}